model/src/main/kotlin/util/FindingsMatcher.kt

```diff
@@ -166,6 +166,6 @@ class FindingsMatcher(
                 locationsForLicenses[license] ?: sortedSetOf(),
                 copyrightsForLicenses[license] ?: sortedSetOf()
             )
-        }.toSortedSet()
+        }.toSet()
```

Comment:
You could take the chance and switch to `mapTo(mutableSetOf())` to avoid creating a temporary collection.

Reply:
done.